images/Dockerfile

```diff
@@ -1,20 +1,20 @@
 FROM cockpit/infra-base
 LABEL maintainer="'Stef Walter' <stefw@redhat.com>"
 
-RUN dnf -y update && dnf -y install nginx && dnf clean all
-
-RUN mkdir -p /usr/local/bin && \
+RUN dnf -y update && \
+    dnf -y install nginx && \
+    dnf clean all && \
+    mkdir -p /usr/local/bin && \
     ln -sf /dev/stdout /var/log/nginx/access.log && \
     ln -sf /dev/stderr /var/log/nginx/error.log && \
     rm -rf /usr/share/nginx/html && \
     ln -snf /cache/images /usr/share/nginx/html && \
     chown -R user /home/user
 
-ADD install-service /usr/local/bin/
-ADD nginx.conf /etc/nginx/
+COPY install-service /usr/local/bin/
+COPY nginx.conf /etc/nginx/
```

Comment:
Can you please add some explanation to the commit message why COPY is preferrable over ADD? The [documentation](https://docs.docker.com/engine/reference/builder/#add) does not discuss that.

Reply:
I can. COPY is more straightforward -- it only operates on files, while ADD performs magic (downloads files, unpacks archives), therefore it's preferred to use COPY and use ADD only when needed.